cli/createTemplate.sh

```diff
@@ -169,8 +173,8 @@ function options() {
   # Specific input control for mock input
   if [[ "${GENERATION_INPUT_SOURCE}" == "mock" ]]; then
 
-    export CACHE_DIR="${GENERATION_BASE_DIR}/.cache"
-    mkdir -p "${CACHE_DIR}"
+    # Cache for asssembled components
+    export GENERATION_CACHE_DIR="$( getCacheDir "${GENERATION_CACHE_DIR}" "" )"
```

Comment:
Is this needed given L131?

Reply:
Yeah composite input source vs mock input source
If its a composite input then the cache dir is based on the root dir setting, if its a mock input source create a random temp file since we don't really need to cache the contents and can't say when the mock input is shared or not